src/main/res/menu/profile_common.xml

```diff
@@ -8,7 +8,8 @@
 
     <item android:title="@string/global_menu_edit_desktop"
           android:id="@+id/edit_name"
-          app:showAsAction="never"/>
+          android:icon="@drawable/ic_create_white_24dp"
+          app:showAsAction="always"/>
```

Comment:
was the edit pencil/action shown only for profiles and not for groups intentionally? or could I also make it available in a subsequent PR?